servers/rendering/rasterizer_rd/shaders/tonemap.glsl

```diff
@@ -257,7 +257,8 @@ vec3 apply_bcs(vec3 color, vec3 bcs) {
 }
 
 vec3 apply_color_correction(vec3 color, sampler3D correction_tex) {
-	return texture(correction_tex, color).rgb;
+	vec3 uv_interp = color;
```

Comment:
We usually use uv_interp for the uv that comes from the vertex shader. I think we can just eliminate this line altogether.